inc/functions-cookies.php

```diff
@@ -0,0 +1,75 @@
+<?php
+// These functions are going to be moved into the plugin
+// upon cross site roll-out
+
+function delete_GA_cookies() {
```

Comment:
What would be your view on changing this to be a generic function called `delete_cookies()` which expected an array of cookies (rather than having them in place). I'm just imagining a circumstance where we might also want to delete a non-GA cookie

Reply:
OK. I understand that there's an issue with passing arguments to hooked functions within WordPress